.github/workflows/_rocm-test.yml

```
@@ -137,10 +137,16 @@ jobs:
             -w /var/lib/jenkins/workspace \
             "${DOCKER_IMAGE}"
           )
+          # save container name for later step
+          echo "CONTAINER_NAME=${container_name}" >> $GITHUB_ENV
```

```suggestion
          echo "CONTAINER_NAME=${container_name}" >> "$GITHUB_ENV"
```

To appease the lint gods